### PR TITLE
Update sharing-images-in-the-cloud-control-panel.md

### PR DESCRIPTION
**Pull request by 0x91**
Removed outdated information about ORD

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8affbcf050/how-to/).

**Comment by 0x91** [comment]
fixes #2456

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-518dd2e904/how-to/)

**Comment by catlook** [comment]
@0x91 Can you confirm that the rest of the article is complete, correct, and up to date? If so, I'll complete a full audit and run it past the editor.  If you can't, I'll merge this and schedule a full audit for a future sprint.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-518dd2e904/how-to/). :bow:

**Comment by cyrichardson** [comment]
Here's here: 
[sharing-images-in-the-cloud-control-panel_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/1699613/sharing-images-in-the-cloud-control-panel_cyrEdits.docx)

**Comment by cyrichardson** [comment]
@catlook ^^

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-bbf747b9ea/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-6eb1bc8bfe/how-to/)

**Comment by catlook** [comment]
Honsou Dansou has SME approved content.

File: content/cloud-images/sharing-images-in-the-cloud-control-panel.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
 permalink: sharing-images-in-the-cloud-control-panel/
-audit_date:
+audit_date: '2018-02-06'
 title: Sharing images in the Cloud Control Panel
 type: article
 created_date: '2015-01-29'
 created_by: Cloud Images
```
Comment (DonNateR) [inline content/cloud-images/sharing-images-in-the-cloud-control-panel.md]: Switch to "Rackspace Support".

Comment (catlook) [reply content/cloud-images/sharing-images-in-the-cloud-control-panel.md]: doh!

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-4b34e80317/how-to/). :bow:

**Pull request merged**